s3/pkg/exception/exception.go

```diff
@@ -39,7 +39,7 @@ var BucketAlreadyExists = S3Error{Code: 409, Description: "The requested bucket
 
 var NoSuchBackend = S3Error{Code: 404, Description: "The specified backend does not exists."}
 var NoSuchType = S3Error{Code: 404, Description: "The specified backend type does not exists."}
-var BucketDeleteError = S3Error{Code: 500, Description: "The bucket can not be deleted. please delete object first"}
+var BucketDeleteError = S3Error{Code: 500, Description: "The bucket with objects can not be deleted, please delete objects first."}
```

Comment:
please relook at the error code...500 is usually for an _internal server error_ which is NOT the case here (the bucket being deleted has objects in it)

Reply:
ok, good suggestion.